modules/apps/transfer/keeper/msg_server.go

```diff
@@ -55,6 +55,7 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 
 	// if a channel exists with source channel, then use IBC V1 protocol
 	// otherwise use IBC V2 protocol
+	// TODO: Change this
 	channel, isIBCV1 := k.channelKeeper.GetChannel(ctx, msg.SourcePort, msg.SourceChannel)
```

Comment:
At the moment, the Transfer endpoint can't use IBC v2 with a v1 channelID. This is easy to change but pivoting to EVM before doing this